File: Daily Challanges/2026-07-July/2026-07-07.py
def round_to_nearest_multiple(n, m):
    lower = (n // m) * m
    upper = lower + m
    if n - lower < upper - n:
        return lower
    return upper


print(round_to_nearest_multiple(5, 3))
print(round_to_nearest_multiple(17, 4))
print(round_to_nearest_multiple(43, 5))
print(round_to_nearest_multiple(38, 11))
print(round_to_nearest_multiple(93, 12))

File: Daily Challanges/2026-07-July/2026-07-07.js
function roundToNearestMultiple(n, m) {
  const lower = Math.floor(n / m) * m;
  const upper = lower + m;
  if (n - lower < upper - n) {
    return lower;
  }
  return upper;
}

console.log(roundToNearestMultiple(5, 3));
console.log(roundToNearestMultiple(17, 4));
console.log(roundToNearestMultiple(43, 5));
console.log(roundToNearestMultiple(38, 11));
console.log(roundToNearestMultiple(93, 12));
